src/test/kotlin/io/github/anki/anki/controller/AuthControllerTest.kt

```diff
@@ -190,15 +192,24 @@ class AuthControllerTest @Autowired constructor(
                 .exchange()
     }
     companion object {
-
         @Container
         @Suppress("PropertyName")
         private val mongoDBContainer: MongoDBContainer = TestContainersFactory.newMongoContainer()
 
+        @Container
+        @Suppress("PropertyName")
```

Comment:
what is rule you violated?

Reply:
Property name should use the screaming snake case notation when the value can not be changed [PropertyName]
I just dont want to use screaming snake case here

Reply:
it is not a constant, so here should be screaming snake case. Check if it will be worked without